content/tutorial/tutorial.md

```diff
@@ -12,185 +12,189 @@ redirect_from:
 
-## Before We Start the Tutorial {#before-we-start-the-tutorial}
+## ट्यूटोरियल शुरू करने से पहले {#before-we-start-the-tutorial}
 
-We will build a small game during this tutorial. **You might be tempted to skip it because you're not building games -- but give it a chance.** The techniques you'll learn in the tutorial are fundamental to building any React apps, and mastering it will give you a deep understanding of React.
+हम इस ट्यूटोरियल के दौरान एक छोटा गेम बनाएंगे। **आप इसे छोड़ने के लिए उत्सुक होंगे क्यूंकि आप गेम नहीं बना रहे हैं -- लेकिन इसे एक मौका दें।** ट्यूटोरियल में आपके द्वारा सीखी जाने वाली तकनीकें किसी भी React ऐप्स के निर्माण के लिए मौलिक हैं, और इसमें महारत हासिल करने से आपको React की गहरी समझ मिलेगी।
 
->Tip
+>ध्यान दें
 >
->This tutorial is designed for people who prefer to **learn by doing**. If you prefer learning concepts from the ground up, check out our [step-by-step guide](/docs/hello-world.html). You might find this tutorial and the guide complementary to each other.
+>यह ट्यूटोरियल उन लोगों के लिए बनाया गया है, जो **चीज़ें करके सीखना** पसंद करते हैं। यदि आप नीचे से ऊपर कॉन्सेप्ट्स को सीखना पसंद करते हैं, तो हमारे [कस्टेप-बाय-स्टेप गाइड](/docs/hello-world.html) देखें। आपको यह ट्यूटोरियल और गाइड एक-दूसरे के संपूरक लग सकते हैं।
 
-The tutorial is divided into several sections:
+ट्यूटोरियल को कई खंडों में विभाजित किया गया है:
 
-* [Setup for the Tutorial](#setup-for-the-tutorial) will give you **a starting point** to follow the tutorial.
-* [Overview](#overview) will teach you **the fundamentals** of React: components, props, and state.
-* [Completing the Game](#completing-the-game) will teach you **the most common techniques** in React development.
-* [Adding Time Travel](#adding-time-travel) will give you **a deeper insight** into the unique strengths of React.
+* [ट्यूटोरियल के लिए सेटअप](#setup-for-the-tutorial) आपको ट्यूटोरियल का पालन करने के लिए **शुरुआत** देगा।
+* [अवलोकन](#overview) आपको React की **बुनियादी बातें**: कौम्पोनॅन्ट्स, props और स्टेट सिखाएगा।
+* [गेम को पूरा करना](#completing-the-game) आपको React विकास में **सबसे आम तकनीकों** को सिखाएगा।
+* [टाइम ट्रैवल को गेम में डालना](#adding-time-travel) से आपको React की अद्वितीय शक्तियों के बारे में गहराई से जानकारी मिलेगी।
 
-You don't have to complete all of the sections at once to get the value out of this tutorial. Try to get as far as you can -- even if it's one or two sections.
+इस ट्यूटोरियल से मूल्य प्राप्त करने के लिए आपको एक बार में सभी वर्गों को पूरा करने की आवश्यकता नहीं है।
+जहां तक ​​हो सके जाने की कोशिश करें -- भले ही यह एक या दो सेक्शन हो।
 
-It's fine to copy and paste code as you're following along the tutorial, but we recommend to type it by hand. This will help you develop a muscle memory and a stronger understanding.
+ट्यूटोरियल के साथ कोड का कॉपी और पेस्ट करना ठीक है, लेकिन हम इसे हाथ से टाइप करने की सलाह दें देते हैं। इससे आपको मांसपेशियों की याददाश्त और मजबूत समझ विकसित करने में मदद मिलेगी।
```

Comment:
कोड "ko" कॉपी

Reply:
हाथ से टाइप करने की सलाह dete hain, de is extra

Reply:
इससे आपको मांसपेशियों की याददाश्त और मजबूत समझ विकसित करने में मदद मिलेगी। -> grammar check